feat: tldr for choosing integration method (#4528)
JetBrains/kotlin-web-site: docs/topics/multiplatform/native-spm.md

[//]: # (title: Swift package export setup)

<tldr>

>    This is a remote integration method that can work for you if:<br/>

Reviewer: ```suggestion
   This local integration method can work for you if:<br/>
```

>    * You want to separate the code base of your final application from the common code base.

Reviewer: ```suggestion
   * You want to separate the codebase of your final application from the common codebase.
```

As far as I can see there is no one true spelling, but elsewhere on kotlinlang.org we use this as a one word term.
